src/languages/python.js

```diff
@@ -383,6 +383,39 @@ export default function(hljs) {
         relevance: 0
       },
       { match: /\bor\b/, scope: "keyword" },
+      // Method calls with parentheses
+      {
+        match: [
+          /\./,
+          IDENT_RE,
+          /(?=\s*\()/
+        ],
+        scope: {
+          2: "title.function.method"
+        }
+      },
+      // Chained method calls
+      {
+        match: [
+          /\./,
+          IDENT_RE,
+          /(?=\s*\.\s*\w)/
+        ],
+        scope: {
+          2: "title.function.method"
+        }
+      },
+      {
+        match: [
+          /(?<!(?:import|from)\s+[\w.]*)/,  // Negative lookbehind for import context
```

Comment:
Do we have a test case for this?  I'd like to see an example or two of what exactly we're avoiding here.

Reply:
Imagine the following line:

```python
from base.derive import test
```

I wouldn't want `derive` to be highlighted as a property.